resources/user.rb

```diff
@@ -21,6 +21,9 @@
 
 unified_mode true
 
+deprecated 'The `grafana_user` resource will be removed in the next major version of the grafana cookbook. ' \
+  'Use e.g. Terraform to interact with the Grafana API instead.'
```

Comment:
This doesn't read right to me, maybe something like below would be better?
```suggestion
  'Use another tool such as Terraform to interact with the Grafana API instead.'
```